### PR TITLE
Rice

### PR DESCRIPTION
**Pull request by SirMacke**

### CODE REVIEW

**Comment by SirMacke** [comment]
...

**Comment by fogge1** [comment]
veri bed kode, i cant not accepting this kode

File: index.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
   <title>githubTest</title>
 </head>
 <body>
-  
+  <p>Rice</p>
```
Comment (isafog) [inline index.html:9]: You cant write p tag u need h1 tag

**Review by NielsHouben — APPROVED** [review]
looks gret

**Review by fogge1 — APPROVED** [review]

**Pull request merged**